README.md

```diff
@@ -89,7 +89,7 @@ The `distro` package implements a robust and inclusive way of retrieving the
 information about a distribution based on new standards and old methods,
 namely from these data sources (from high to low precedence):
 
-* The os-release file `/etc/os-release`, if present.
+* The os-release file `/etc/os-release` if present, with a fall-back on `/usr/lib/os-release` if needed.
```

Comment:
I wonder if it makes sense to link to https://www.freedesktop.org/software/systemd/man/os-release.html here or somewhere?

Reply:
Sounds good, but seems unrelated to this pull request.

Reply:
Not sure how you think it's unrelated, this was the reference given in the PR body?

Reply:
IMHO the documentation needs a global update. Maybe we could address this in another PR ?